packages/sonarwhal/src/lib/cli/init.ts

```diff
@@ -33,14 +76,13 @@ export const initSonarwhalrc = async (options: CLIOptions): Promise<boolean> =>
 
     const connectorKeys: Array<inquirer.ChoiceType> = resourceLoader.getCoreConnectors().concat(resourceLoader.getInstalledConnectors());
     const formattersKeys: Array<inquirer.ChoiceType> = resourceLoader.getCoreFormatters();
-    const rulesIds = resourceLoader.getCoreRules();
-    const rulesConfig = rulesIds.reduce((config, ruleId) => {
-        config[ruleId] = 'warning';
-
-        return config;
-    }, {});
-
-    const rules = resourceLoader.loadRules(rulesConfig);
+    const npmRules: Array<NpmPackage> = await resourceLoader.getCoreRulesFromNpm();
+    const rules = npmRules.map((rule) => {
```

Comment:
We need to find a way to know which ones are recommended or not for the defaults.
E.g.: `amp-validator`, `cloudinary`, etc. shouldn't be on by default.

Do we have access to the keywords? Maybe we can use that to filter?

Reply:
yep, we can read the keywords, which one do you want to use for that?

Reply:
`recommended`, `sonarwhal-recommended` or something like that. @alrra?

Reply:
@alrra @molant done!

Reply:
For the record, we decided to go with `sonarwhal-recommended` so it's more scoped.